netty/src/main/java/io/grpc/netty/NettyServerBuilder.java

```diff
@@ -247,6 +249,18 @@ protected NettyServer buildTransportServer() {
   }
 
   @Override
+  public NettyServerBuilder useTransportSecurity(TrustManagerFactory trustManagerFactory,
+                                                 KeyManagerFactory keyManagerFactory) {
+    try {
+      sslContext = GrpcSslContexts.forServer(trustManagerFactory, keyManagerFactory).build();
+    } catch (SSLException e) {
+      // This should likely be some other, easier to catch exception.
```

Comment:
Since you're not passing `File` objects, you don't even need to catch and re-throw.

As far as a better wrapper exception type for the other method, perhaps `IllegalArgumentException`, since it indicates that a specified file is not a valid key or truststore?